assets/sass/main.scss

```diff
@@ -22,4 +22,5 @@ $brand-primary: #005EB8;
 /*!
  * A+ Styles
  */
-@import 'protocols/quiz1';
+ @import 'protocols/quiz1';
```

Comment:
Why extra space in the start of the line?

Reply:
Because this PR is. a draft and there are still things to tweak, code style included.